documentation/getting-started/getting-started-kotlin.asciidoc

```diff
@@ -0,0 +1,164 @@
+---
+title: Using Vaadin with Kotlin
+order: 600
+layout: page
+---
+
+= Using Vaadin with Kotlin
+
+Vaadin supports the Kotlin programming language through the Karibu-DSL library, which contains various extensions to use Vaadin with Kotlin.
+Perhaps most importantly, the library allows you to create your Vaadin UI in a declarative way using a _domain-specific language_ (DSL).
+These features are enabled by the http://www.vaadinonkotlin.eu/index.html[Vaadin on Kotlin] project.
+
+For a quick introduction to the DSL, to have a form with two text input fields, you define them hierarchically under the `init` method (a block itself) as follows:
+
+----
+@Route("")
+class MyView : VerticalLayout() { <1>
+    init {
+        formLayout { <2>
+            textField("Name:") { <3>
+                description = "Last name, first name" <4>
+            }
+            textField("Age:") {
+                width = "5em"
+            }
+        }
+    }
+}
+----
+<1> A view should always extend a Vaadin layout component.
```

Comment:
If we decide to go with `KComposite` then we need to change this to "A view always extends KComposite".